.bazelrc

```diff
@@ -23,6 +23,11 @@ build --tool_java_language_version=9
 build:windows --extra_toolchains=@local_config_cc//:cc-toolchain-x64_windows-clang-cl
 build:windows --extra_execution_platforms=//:x64_windows-clang-cl
 build:windows --features=static_link_msvcrt
+# Required so that the JDK can find itself when loading the instrument agent lib.
+# FIXME: Debug why this is the case even though sun.boot.library.path is correctly set to
+#        external\local_jdk\bin under the Bazel cache.
+test:windows --noincompatible_strict_action_env
```

Comment:
That is true, but if that happens we will simply have to maintain our own value for `PATH`. Note that the fix here is to *disable* this option which we enabled above, so it will continue to work after the flag has been removed.